drivers/sensor/ti_hdc/ti_hdc.h

```diff
@@ -23,14 +23,14 @@
 
 struct ti_hdc_config {
 	struct i2c_dt_spec i2c;
+	struct gpio_dt_spec drdy;
 };
 
 struct ti_hdc_data {
 	uint16_t t_sample;
 	uint16_t rh_sample;
 
 #if DT_INST_NODE_HAS_PROP(0, drdy_gpios)
```

Comment:
this is now uncoditional, it wasn't before, also, `drdy` is not initialized

Reply:
Fixed.